pkg/cli/image/mirror/mirror.go

```diff
@@ -743,11 +744,21 @@ func copyBlob(ctx context.Context, plan *workPlan, c *repositoryBlobCopy, blob d
 		}
 		plan.BytesCopied(n)
 		return nil
-	}()
-	if err != nil {
-		return err
 	}
-	return nil
+
+	var err error
+	for i := 0; i < 5; i++ {
+		if err = copyfn(); err != nil {
+			if strings.Contains(err.Error(), "REFUSED_STREAM") {
+				klog.V(6).Infof("retrying push due to %s", err.Error())
+				time.Sleep(time.Second)
```

Comment:
How about using `k8s.io/client-go/util/retry/util#OnError` or at least `k8s.io/client-go/util/flowcontrol#Backoff`?

Reply:
Yes, that would be neat. Done.